### PR TITLE
Add the color of the node property

### PR DESCRIPTION
**Pull request by comadrejo**
Gets and sets the color value of the little box on the right of the node in the node view

### CODE REVIEW

File: openHarmony/openHarmony_node.js [hunk 1]
```diff
@@ -368,6 +368,22 @@ Object.defineProperty($.oNode.prototype, 'name', {
   }
 });
 
+/**
+ * The node's color.
+ * @name $.oNode#nodeColor
+ * @type {$.oColorValue}
+ */
+ Object.defineProperty($.oNode.prototype, 'nodeColor', {
```
Comment (mchaptel) [inline openHarmony/openHarmony_node.js:376]: how would you feel about just naming it oNode.color ? oNode.nodeColor seems redundant?

File: openHarmony/openHarmony_node.js [hunk 2]
```diff
@@ -368,6 +368,22 @@ Object.defineProperty($.oNode.prototype, 'name', {
...
+  get : function(){
+    var _nodecolor = node.getColor(this.path)
```
Comment (mchaptel) [inline openHarmony/openHarmony_node.js:378]: I know they're optional but for consistency's sake, would you mind adding the ";" at the end of the lines?

**Review by mchaptel — CHANGES_REQUESTED** [review]
that's a really good idea, thank you for adding it! I've got a couple of discussions before merging but it's not much

File: openHarmony/openHarmony_node.js (hunk 1 above)
Comment (comadrejo) [reply openHarmony/openHarmony_node.js]: I agree. I just updated the file by making another commit, I'm not sure if I it is the right way to do it

**Comment by mchaptel** [comment]
thanks for this addition!!

File: openHarmony/openHarmony_node.js (hunk 1 above)
Comment (comadrejo) [reply openHarmony/openHarmony_node.js]: I just found that the Colour Card nodes have the ["color"](https://cfourney.github.io/OpenHarmony/NodeTypes.html#Colour-Card) property which will cause conflict now that we named this oNode.color
What should we name this then? littleBoxInNodeViewColor? 🤣

Comment (mchaptel) [reply openHarmony/openHarmony_node.js]: Agh, let's just go back to nodeColor! We do have nodePosition after all. Sorry about that

Comment (mchaptel) [reply openHarmony/openHarmony_node.js]: I've reverted the property name to nodeColor, it's in the latest master @comadrejo

Comment (comadrejo) [reply openHarmony/openHarmony_node.js]: Great news :). Sorry that I forgot about this hehe

**Pull request merged**